src/main/app/src/app/zoeken/zoek/zoek.component.ts

```diff
@@ -20,36 +20,28 @@ import {ZoekResultaat} from '../model/zoek-resultaat';
 import {ZoekType} from '../model/zoek-type';
 import {DocumentZoekObject} from '../model/documenten/document-zoek-object';
 import {ZoekObjectType} from '../model/zoek-object-type';
+import {ZoekVeld} from '../model/zoek-veld';
 
 @Component({
     selector: 'zac-zoeken',
     templateUrl: './zoek.component.html',
     styleUrls: ['./zoek.component.less']
 })
-export class ZoekComponent implements AfterViewInit {
+export class ZoekComponent implements AfterViewInit, OnInit {
 
     @ViewChild('paginator') paginator: MatPaginator;
-    @Input() sideNav: MatSidenav;
-
-    @Input() set keywords(value: string) {
-        if (this.zoekenControl.value !== value) {
-            this.zoekenControl.setValue(value);
-            this.zoek.emit();
-        }
-    }
-
-    get keywords(): string {
-        return this.zoekenControl.value;
-    }
+    @Input() zoekenSideNav: MatSidenav;
 
     zoekType: ZoekType = ZoekType.ZAC;
     ZoekType = ZoekType;
+    ZoekVeld = ZoekVeld;
```

Comment:
En hier?

Reply:
Hier wordt een enum aan een variable toegekend zodat deze in de html kan worden gebruikt.
Deze heeft een hoofdletter zodat je weet dat het een enum is.